.github/workflows/nix.yaml

```
@@ -99,23 +77,93 @@ jobs:
+  docker-build-amd64:
+    needs: [build]
+    runs-on: x86_64-linux
+    outputs:
+      image-name: ${{ steps.image-info.outputs.name }}
+      image-tag: ${{ steps.image-info.outputs.tag }}
+    steps:
+      - uses: actions/checkout@v4
+      
+      - name: Get image info
+        id: image-info
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:58: Double quote to prevent globbing and word splitting

(shellcheck)